AnimatedComponents/src/components/slider/range-slider/RangeSlider.tsx

```diff
@@ -63,16 +69,24 @@ const RangeSlider = (props: IRangeSliderProps) => {
     return newMaxValue
   }, [max, min, step, sliderWidth, positionUpperLimit.value])
 
+  const setLowerToolTipText = (value: string) => {
+    lowerToolTipRef.current?.setNativeProps({ text: value })
```

Comment:
check if optional chaining is not required

Reply:
it's because initial value is null